### PR TITLE
Document NU1504, NU1505, NU1506

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/2700

### CODE REVIEW

File: docs/reference/errors-and-warnings/NU1506.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<pre>Duplicate 'PackageVersion' items found. Remove the duplicate items or use the Update functionality to ensure a consistent restore behavior. The duplicate 'PackageVersion' items are: X [1.0.0], X [2.0.0].</pre>
+
+### Issue
+
+Sometimes when adding `PackageVersion` items, you may end up adding an item for a particular package multiple times.
+
+Example:
+
+```xml
+
```
Comment (dominoFire) [inline docs/reference/errors-and-warnings/NU1506.md:25]: nit: extra line. This applies to all three  doc. pages.

File: docs/reference/errors-and-warnings/NU1506.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# NuGet Error NU1506
+
+## Scenario 1
+
+<pre>Duplicate 'PackageVersion' items found. Remove the duplicate items or use the Update functionality to ensure a consistent restore behavior. The duplicate 'PackageVersion' items are: X [1.0.0], X [2.0.0].</pre>
+
+### Issue
+
+Sometimes when adding `PackageVersion` items, you may end up adding an item for a particular package multiple times.
```
Comment (dominoFire) [inline docs/reference/errors-and-warnings/NU1506.md:20]: Is `<PackageVersion />` deprecated ? https://github.com/Microsoft/MSBuildSdks/tree/main/src/CentralPackageVersions#version-20-breaking-changes 

If so, should we write that in this documentation page?

File: docs/reference/errors-and-warnings/NU1504.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+# NuGet Error NU1504
+
+## Scenario 1
+
+<pre>Duplicate 'PackageReference' items found. Remove the duplicate items or use the Update functionality to ensure a consistent restore behavior. The duplicate 'PackageReference' items are: X 1.0.0, X 2.0.0.</pre>
+
+### Issue
+
+Sometimes when adding `PackageReference` items, you may end up adding an item for a particular package multiple times.
```
Comment (dominoFire) [inline docs/reference/errors-and-warnings/NU1504.md:20]: I tried to be more concise. Here's my suggestion:

```suggestion
This error is raised when adding multiple `PackageReference` items, in the same or in multiple project files.
```

**Review by dominoFire — APPROVED** [review]
Some comments on PackageVersion.

File: docs/reference/errors-and-warnings/NU1506.md (hunk 2 above)
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU1506.md:20]: You are conflating NuGet's native CPM implementation and the MSBuild SDK one. 
PackageVersion is very much not deprecated in NuGet's CPM.

File: docs/reference/errors-and-warnings/NU1506.md (hunk 1 above)
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU1506.md:25]: Nice catch. 
Fixed everywhere.

File: docs/reference/errors-and-warnings/NU1504.md (hunk 3 above)
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU1504.md:20]: I appreciate the suggestion, but that text would effective duplicate the scenario message. 

The description is meant to explain how you may have run into the scenario.

**Pull request merged**